packages/hint-compat-api/src/helpers/compat-css.ts

```diff
@@ -26,7 +26,7 @@ export class CompatCSS extends CompatBase<StyleEvents, StyleParse> {
     }
 
     public searchFeatures(parse: StyleParse): void {
-        this.walk(parse.ast);
+        this.walk(parse.ast, parse.code);
```

Comment:
I think we want something narrower than the entire CSS file. We should either omit this for CSS or only include code which is near the problem. 

If we want to serialize something custom I think having the selector plus the relevant CSS property/value (ignoring all other surrounding properties) would be good.

@webhintio/contributors Thoughts?

Reply:
Definitely, somehow I thought `walk` did this automatically and it was just a feature we weren't using 😖

Reply:
I thought `pase.code` had only the feature hehehe. I will look into it.

Reply:
done!